build_tools/benchmarks/benchmark_helper.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Miscellaneous tool to help work with benchmark suite and benchmark CI."""
+
+import pathlib
+import sys
+
+# Add build_tools python dir to the search path.
+sys.path.insert(0, str(pathlib.Path(__file__).parent.with_name("python")))
+
+import argparse
+import json
+from typing import List
+
+from e2e_test_framework import serialization
+from e2e_test_artifacts import model_artifacts, iree_artifacts
+from e2e_test_framework.definitions import iree_definitions
+
+IREE_COMPILER_NAME = "iree-compile"
+
+
+def _dump_cmds_of_generation_config(
+    gen_config: iree_definitions.ModuleGenerationConfig,
+    root_path: pathlib.PurePath = pathlib.PurePath()):
+
+  imported_model = gen_config.imported_model
+  imported_model_path = iree_artifacts.get_imported_model_path(
+      imported_model=imported_model, root_path=root_path)
+  module_path = iree_artifacts.get_module_dir_path(
+      module_generation_config=gen_config,
+      root_path=root_path) / iree_artifacts.MODULE_FILENAME
+  compile_cmds = [
+      IREE_COMPILER_NAME,
+      str(imported_model_path), "-o",
+      str(module_path)
+  ]
+  compile_cmds += gen_config.materialize_compile_flags()
+
+  if imported_model.import_config.tool == iree_definitions.ImportTool.NONE:
+    import_cmds = ["(Source model is already in MLIR)"]
```

Comment:
Maybe make this have a bash comment to make copy-paste easier?

```suggestion
    import_cmds = ["# (Source model is already in MLIR)"]
```